### PR TITLE
Blog section update 

### PR DESCRIPTION
**Pull request by SafeEHA**
Changed the position and style of the blog section

### CODE REVIEW

File: _sass/_homepage.scss [hunk 1]
```diff
@@ -109,33 +109,57 @@
...
         position: relative;
         -webkit-overflow-scrolling: touch;
-
+        @media (max-width: $mk--breakpoint--md) {
+          display: block;
+          width: 100%;
+        }
         @media (min-width: $mk--breakpoint--md) {
-            width: auto;
+          width: auto;
```
Comment (elfosardo) [inline _sass/_homepage.scss:134]: remove indentation change

**Review by elfosardo — CHANGES_REQUESTED** [review]
please remove all the indentation or code style changes

File: index.html [hunk 2]
```diff
@@ -33,6 +33,29 @@ <h2 class="mk-heading mk-heading--xl mk-m-border">Why Metal3.io?</h2>
           <p>The Metal3.io project is also building integration with the Kubernetes cluster-api project, allowing
           Metal3.io to be used as an infrastructure backend for Machine objects from the Cluster API.</p>
 </section>
+<section class="mk-blog-baremetal mk-main__section">
+    <header class="mk-main__header">
+      <h2 class="mk-heading mk-heading--xl mk-m-border">What's new?</h2>
+    </header>
+    <p class="mk-heading--lg">A hub for news, updates, and insights on Metal3 and its applications, as well as tips and best practices for using the platform.</p>
+    <div class="mk-blog-meta__preview__scroll-wrapper">
```
Comment (hroyrh) [inline index.html:41]: Can you add a single line space before the div block

**Pull request merged**